packages/control-property-editor-common/src/api.ts

```diff
@@ -106,6 +110,7 @@ export interface OutlineNode {
     children: OutlineNode[];
     icon?: string;
     hasDefaultContent?: boolean;
+    fe?: boolean;
```

Comment:
The UI should be as simple as possible and it shouldn't contain UI5 specific information.

The node filtering should happen in the `preview-middleware-client` when the outline nodes from UI5 are converted.

Reply:
Thanks. Good point.